src/assets/i18n/en.json

```diff
@@ -1,22 +1,45 @@
 {
-"TAB": {  
-"LEARNING" : "Learning Resources",
-"OBSERVATION": "Observations",
-"ASSESMENTS": "Assesments",
-"IMPROVEMENT": "Improvement Projects"
-},
-"BUTTONS": {
-    "APPLY": "Apply",
-    "CLEAR": "Clear",
-    "VIEWMORE": "View More"
-},
-"SIDEMENU": {
-    "LIBRARY": "Library",
-    "MYFOLDER": "My Folder"
-},
-"SIDE_1": "side1",
-"SIDE_2": "side2",
-"SIDE_3": "side3"
-}
-
-
+    "TAB": {
```

Comment:
As per yesterdays call we discussed to have only 3 outside object
ACTIONS, MESSAGES, TITLES. Please make the necessary changes

Reply:
Modified